tests/monitoring/monitoring.go

```diff
@@ -124,21 +126,9 @@ var _ = Describe("[Serial][sig-monitoring]Monitoring", Serial, decorators.SigMon
 			originalKv := util.GetCurrentKv(virtClient)
 			kv, err := virtClient.KubeVirt(originalKv.Namespace).Get(context.Background(), originalKv.Name, metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
-			kv.Spec.CustomizeComponents = v1.CustomizeComponents{
-				Patches: []v1.CustomizeComponentsPatch{
-					{
-						ResourceName: virtHandler.deploymentName,
-						ResourceType: "DaemonSet",
-						Patch:        `{"spec":{"template":{"spec":{"nodeSelector":{"kubernetes.io/hostname":"does-not-exist"}}}}}`,
-						Type:         v1.StrategicMergePatchType,
-					},
-				},
-			}
 
-			Eventually(func() error {
-				kv, err = virtClient.KubeVirt(originalKv.Namespace).Update(context.Background(), kv, metav1.UpdateOptions{})
-				return err
-			}, 30*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
+			_, err = patch.New(patch.WithAdd("/spec/template/spec/nodeSelector/kubernetes.io~1hostname", "does-not-exist")).GeneratePayload()
```

Comment:
It should return patch bytes (`patchBytes, err = ...`) so then you can send in `PATCH` (HTTP request) with this actual data. Very similar to what you did below:
```
patchBytes, err := patch.New(patch.WithRemove("/spec/customizeComponents")).GeneratePayload()
Expect(err).ToNot(HaveOccurred())

_, err = virtClient.KubeVirt(originalKv.Namespace).Patch(context.Background(), originalKv.Name, types.JSONPatchType, patchBytes, metav1.PatchOptions{})
Expect(err).ToNot(HaveOccurred())
```
Here `patchBytes` is used when you do `Patch` API request, instead `Update`.